### PR TITLE
Fix undefined reconnect delay in client

### PR DESCRIPTION
**Pull request by katkad**
"Connection to back-end lost :-( Retrying in undefineds..."

### CODE REVIEW

**Comment by fw42** [comment]
This will reset the delay to 1 on every single try, i.e., it will never increase.

**Comment by fw42** [comment]
You can set it in the constructor so it will only be defined once at the beginning, that should work.

**Comment by fw42** [comment]
Also, your clock is 4 hours in the future...

```
 commit af4f02cf7ab794e82627243fd6a63e9360f33b57
 Author: katkad <katarina.durechova@nic.cz>
 Date:   Mon Sep 16 12:52:03 2013 -0400
```

```
>> Time.now
=> 2013-09-16 09:08:16 -0400
```

**Comment by katkad** [comment]
right. got wrong timezone. thanks

yeah, it is better in constructor

File: client/coffee/transport.coffee [hunk 1]
```diff
@@ -2,13 +2,13 @@ class Transport
   constructor: (instance, handler, log) ->
     @log = log
     @handler = handler
+    @reconnectDelay = 1
     @connect()
 
   connect: ->
     @socket = new SockJS('/data')
```
Comment (fw42) [inline client/coffee/transport.coffee:9]: This needs to stay or the reset to 1s after successful connect wont work.

**Comment by katkad** [comment]
updated

**Comment by fw42** [comment]
Thanks!

**Pull request merged**